extensions/wikia/Recirculation/images/logomark.svg

```diff
@@ -0,0 +1,23 @@
+<svg width="20" height="19" viewBox="0 0 20 19" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We should name it "fandom-logo" or something like that, so it's clear from the name what it is.